package.json

```
@@ -20,7 +20,7 @@
 		"check:type": "npm run check:type --workspaces --if-present",
 		"fix": "npm run prettify && npm run check:lint -- --fix",
 		"prettify": "prettier . --write --ignore-unknown",
-		"test": "npm run test --workspaces --if-present",
+		"test": "npm run clean --workspace=wrangler && npm run bundle --workspace=wrangler && npm run test --workspaces --if-present",
```

```suggestion
		"test": "npm run build --workspace=wrangler && npm run test --workspaces --if-present",
```

wrangler's build runs clean, bundle, and emit types for us 😅

---

We want to avoid generating the types when testing. That is the process that takes up the most time - like 10x the time of bundling.

---

I don't think that any of our e2e tests rely upon the typings, right? They are only really important for publishing.